pages/lib/markdown.js

```diff
@@ -126,14 +141,18 @@ function findPath(context, search) {
 
   for (var ii = 0; ii <= relative.length; ii++) {
     var path = relative.slice(0, relative.length - ii).concat(search);
-    if (path.reduce(
-      (def, name) => def && (
-        (def.module && def.module[name]) ||
-        (def.interface && allMembers && allMembers.get(def.interface)[name]) ||
-        undefined
-      ),
-      {module: defs}
-    )) {
+    if (
+      path.reduce(
+        (def, name) =>
+          def &&
+          ((def.module && def.module[name]) ||
+            (def.interface &&
+              allMembers &&
+              allMembers.get(def.interface)[name]) ||
+            undefined),
+        { module: defs }
```

Comment:
you can use `--no-bracket-spacing` to remove spaces here. It seems like at fb we don't add it there.